docs: update to "Matching one of the two alternative locators" section

Anand-M-Cherian opened the pull request:
The section for "Matching one of the two alternative locators" mentioned "matching all of the alternatives". However, since we are using or() operator, it should match "any one of the alternatives" to locate the element.

![update_locators md_file](https://github.com/user-attachments/assets/721a4b54-4e60-429b-bc92-48547b770f4f)

@playwrightmachine If my understanding is wrong, kindly correct me.

Anand-M-Cherian committed:
Update docs/src/locators.md

The section for "Matching one of the two alternative locators" mentioned "matching all of the alternatives". However, since we are using or() operator, it should match "any one of the alternatives" to locate the element.

Signed-off-by: Anand M Cherian <63868951+Anand-M-Cherian@users.noreply.github.com>

dgozman commented on docs/src/locators.md:
```diff
@@ -1218,7 +1218,7 @@ var button = page.GetByRole(AriaRole.Button).And(page.GetByTitle("Subscribe"));
 ### Matching one of the two alternative locators
 
-If you'd like to target one of the two or more elements, and you don't know which one it will be, use [`method: Locator.or`] to create a locator that matches all of the alternatives.
+If you'd like to target one of the two or more elements, and you don't know which one it will be, use [`method: Locator.or`] to create a locator that matches any one of the alternatives.
```
```suggestion
If you'd like to target one of the two or more elements, and you don't know which one it will be, use [`method: Locator.or`] to create a locator that matches any one or both of the alternatives.
```

Anand-M-Cherian replied:
I think that is an even more comprehensive explanation of the or() operator leaving no room for doubt for an absolute beginner. Let's go ahead with this.

dgozman reviewed (commented):
Thank you for the PR! Since the locator can match either the first, or the second, or both at the same time, I'd suggest to be as explicit as possible. WDYT?

Anand-M-Cherian committed:
Update docs/src/locators.md

The section for "Matching one of the two alternative locators" mentioned "matching all of the alternatives". However, since we are using or() operator, it should match "any one of the alternatives" to locate the element.

As per the suggestion in the review comment, instead of "any one of the alternatives" it was decided to go ahead with "any one or both of the alternatives" to be as clear and explicit as possible.

Co-authored-by: Dmitry Gozman <dgozman@gmail.com>
Signed-off-by: Anand M Cherian <63868951+Anand-M-Cherian@users.noreply.github.com>

dgozman reviewed (approved):
Looks great, merging in!

Anand-M-Cherian commented:
![image](https://github.com/user-attachments/assets/79fe8c01-654c-4e3f-b78b-9c5e600c22b6)
After the changes were deployed....
Thank you @dgozman
